### PR TITLE
Better GraphQL error handling

### LINKED ISSUE #620: [TASK] Enhance the Graphql loading status and Error handling
## Overview
- Enhance the error handling for graphql hooks to cover aloso null/ empty values logic error
- Better handling of loading state with flashing views

### LINKED ISSUE #666: [TASK] Update chain logo icon from Ethereum to Webb
## Overview

In the proposals page for open proposals we are currently displaying the Ethereum logo icon when it should be the webb logo. 

### Screenshot 
Screen Shot 2022-10-27 at 10 23 47 AM

### Task Checklist 

- [ ] Update icon to reflect webb logo

### LINKED ISSUE #669: [BUG] Proposal drawer crashed stats page
## Overview

When you navigate to Proposals > details the stats dapp crashes. We need to cover more edge cases to ensure the resiliency of the stats dapp. 

### Error

CleanShot 2022-10-31 at 21 25 38@2x

### PR DESCRIPTION
**Pull request by AhmedKorim**
## Summary of changes
- Graphql Error handling
	- Utilising  apollo client retry link for retry in case of server-side/network errors
	- Using the apollo client error link for showing messages (The message is pending the adding of Alert component)
- Graphql/UI misc fixes
	- Fixing the latest session, ensure the latest session has a public key 
	- Fix the authorities details button to consume the Validator id instead of the session validator
	- Adding backlink labels for the drawers
	- Remove dead links
	- Change the network signature/keygen  thresholds to be a line chart
	- Misc UI changes from checklist #619 

### Reference issue to close (if applicable)
- Closes #620 
- Closes #669
- Closes #666  

-----
### Code Checklist 

- [ ] Tested
- [ ] Documented

### CODE REVIEW

**Comment by drewstone** [comment]
What's the status of this @AhmedKorim ?

File: apps/stats-dapp/src/containers/AuthorityDetail/AuthorityDetail.tsx [hunk 1]
```diff
@@ -262,7 +270,12 @@ const DetailsView: FC<{
                 className={cx('mr-4 inline-block', { hidden: !isPage })}
                 hidden={!isPage}
               >
-                <ArrowLeft size="lg" />
+                <div className="flex flex-raw items-center">
```
Comment (AtelyPham) [inline apps/stats-dapp/src/containers/AuthorityDetail/AuthorityDetail.tsx:273]: Please fix the typo here `flex-raw`.

File: apps/stats-dapp/src/containers/ProposalsTable/ProposalsTable.tsx [hunk 2]
```diff
@@ -267,100 +273,107 @@ export const ProposalsTable = () => {
...
+            </AccordionItem>
+            <AccordionItem className={'p-4 py-0'} value={'proposal-status'}>
+              <AccordionButton>Proposal Status</AccordionButton>
+              <AccordionContent>
+                <div
+                  style={{
+                    maxWidth: '300px',
+                    maxHeight: 300,
+                    overflow: 'hidden',
+                    overflowY: 'auto',
```
Comment (AtelyPham) [inline apps/stats-dapp/src/containers/ProposalsTable/ProposalsTable.tsx:317]: I'm curious why don't you use tailwind classes for styling here?

File: apps/stats-dapp/src/containers/ProposalsTable/ProposalsTable.tsx [hunk 3]
```diff
@@ -267,100 +273,107 @@ export const ProposalsTable = () => {
...
-                    setSelectedChains(v);
+            <AccordionItem className={'p-4 py-0'} value={'chain'}>
+              <AccordionButton>Chain</AccordionButton>
+              <AccordionContent>
+                <div
+                  style={{
+                    maxWidth: '300px',
+                    maxHeight: 300,
+                    overflow: 'hidden',
+                    overflowY: 'auto',
```
Comment (AtelyPham) [inline apps/stats-dapp/src/containers/ProposalsTable/ProposalsTable.tsx:349]: I'm curious why don't you use tailwind for styling here?

File: apps/stats-dapp/src/containers/ProposalsTable/ProposalsTable.tsx [hunk 4]
```diff
@@ -267,100 +273,107 @@ export const ProposalsTable = () => {
...
+                      setSelectedChains(v);
+                    }}
+                    iconGetter={([_key, chainConfig]) => (
+                      <div
+                        style={{
+                          maxWidth: 20,
+                          maxHeight: 20,
+                          overflow: 'hidden',
+                          backgroundSize: '20px 20px',
+                        }}
```
Comment (AtelyPham) [inline apps/stats-dapp/src/containers/ProposalsTable/ProposalsTable.tsx:365]: I'm curious why don't you use tailwind for styling here?

**Review by AtelyPham — COMMENTED** [review]
Leave some comments, otherwise all looks good to me 👍

**Review by AtelyPham — APPROVED** [review]
LGTM 👍 Great works @AhmedKorim 🔥

**Pull request merged**